<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualControl</class>
 <widget class="QMainWindow" name="ManualControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="title" native="true">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>1091</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 127);
selection-background-color: rgb(0, 0, 0);
alternate-background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>30</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>8514oem</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;8514oem&quot;;
text-decoration: underline;
color: rgb(0, 255, 255);</string>
        </property>
        <property name="text">
         <string>Manual Control</string>
        </property>
       </widget>
      </widget>
      <widget class="QRadioButton" name="highSpeed">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>440</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Perpetua Titling MT</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>High</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="mediumSpeed">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>380</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Perpetua Titling MT</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Medium</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="lowSpeed">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>320</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Perpetua Titling MT</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>low</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>920</x>
         <y>240</y>
         <width>55</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Pristina&quot;;
font: 75 8pt &quot;8514oem&quot;;
font: 75 8pt &quot;8514oem&quot;;
text-decoration: underline;
font: 16pt &quot;Pristina&quot;;
font: 16pt &quot;MS Shell Dlg 2&quot;;
font: 8pt &quot;Pristina&quot;;
font: 16pt &quot;Pristina&quot;;</string>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>200</y>
         <width>391</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 italic 12pt &quot;8514oem&quot;;
font: 75 8pt &quot;8514oem&quot;;</string>
       </property>
       <property name="text">
        <string>Choose how do you want the car to move:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="frontButton">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>240</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>arrows/front arrow.png</normaloff>arrows/front arrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="backButton">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>430</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>arrows/back arrow.png</normaloff>arrows/back arrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="rightButton">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>330</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>arrows/right arrow.png</normaloff>arrows/right arrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="startButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>260</y>
         <width>181</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Pristina&quot;;
font: 16pt &quot;Onyx&quot;;
</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stopButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>400</y>
         <width>181</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Pristina&quot;;
font: 16pt &quot;Onyx&quot;;
</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="leftButton">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>380</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>arrows/left arrow.png</normaloff>arrows/left arrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1103</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
